libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts

```diff
@@ -32,38 +39,38 @@ export function generateCohortsStatsTable(
       let metricMin = Number.MAX_SAFE_INTEGER;
       let metricMax = Number.MIN_SAFE_INTEGER;
       cohorts.forEach((_errorCohort, cohortIndex) => {
-        const stat = labeledStatistics[cohortIndex].find(
-          (stat) => stat.key === metricOption.key
+        const labeledStat = labeledStatistics[cohortIndex].find(
+          (labeledStat) => labeledStat.key === metricOption.key
         );
-        if (stat) {
-          if (stat.stat > metricMax) {
-            metricMax = stat.stat;
+        if (labeledStat) {
+          if (labeledStat.stat > metricMax) {
+            metricMax = labeledStat.stat;
           }
-          if (stat.stat < metricMin) {
-            metricMin = stat.stat;
+          if (labeledStat.stat < metricMin) {
+            metricMin = labeledStat.stat;
           }
         }
       });
       // use min and max to normalize the colors
       const metricMinMaxDiff = metricMax - metricMin;
       cohorts.forEach((_errorCohort, cohortIndex) => {
-        const stat = labeledStatistics[cohortIndex].find(
-          (stat) => stat.key === metricOption.key
+        const labeledStat = labeledStatistics[cohortIndex].find(
+          (labeledStat) => labeledStat.key === metricOption.key
         );
-        if (stat && !Number.isNaN(stat.stat)) {
+        if (labeledStat && !Number.isNaN(labeledStat.stat)) {
           items.push({
-            colorValue: (stat.stat - metricMin) / metricMinMaxDiff,
-            value: Number(stat.stat.toFixed(3)),
+            colorValue: (labeledStat.stat - metricMin) / metricMinMaxDiff,
+            value: Number(labeledStat.stat.toFixed(3)),
             x: metricIndex + 1,
             y: cohortIndex
           });
         } else {
-          // not a numeric value (NaN), so just put null
+          // not a numeric value (NaN), so just put null and use color gray
           items.push({
             color: "#808080",
             value: Number.NaN,

```

Comment:
I wonder if we should use the diagonal lines style of error analysis for NaN cases

Reply:
just something to think about, not necessary for this PR

Reply:
Funny that you'd mention that, our designer said the same thing 😆

Reply:
And yes, this will be in a future PR if I can figure out how to do it with highcharts.